### DISCUSSION TITLE
How to change rotation icon expand all/collapse all

### DISCUSSION THREAD

**Discussion by duyhd** [KevinVandy/material-react-table#816] [General]
I want to edit the rotation of icon expand all or collapse all  of  material-react-table. Can anyone helpe me ?
![Screenshot 2023-11-14 092508](https://github.com/KevinVandy/material-react-table/assets/49530713/676e8882-ff7d-448e-a8cb-f7bdb47c0414)
![Screenshot 2023-11-14 092456](https://github.com/KevinVandy/material-react-table/assets/49530713/872f8e34-4096-4a67-b1f5-138d837bb57e)

**Comment by githorse** [comment]
Hi @duyhd, not sure if it will help but I messed with the expand/collapse-all button like this:

```
  muiExpandAllButtonProps: ({table}) => ({
    sx: {
      svg: {
        transform: `${
	  table.getIsSomeRowsExpanded() ?
	    table.getIsAllRowsExpanded() ?
	      // rotation or other styles here 
        }`,
        transition: 'transform 150ms ease 0s !important'
    }
  }
})
```
```			
icons={{
  ExpandMoreIcon: (props: SvgIconProps) =>
    (<DownArrow {...props} fontSize='small' />),
  KeyboardDoubleArrowDownIcon: (props: SvgIconProps) =>
    (<DoubleDownArrow {...props} fontSize='small' />),
}}
```

**Reply by duyhd** [reply]
I will try it and feedback for u .Thanks
